common/djangoapps/xblock_django/migrations/0002_xblockdisableconfig_deprecated_blocks.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('xblock_django', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='xblockdisableconfig',
+            name='deprecated_blocks',
```

Comment:
I think the tests are failing because the migration file hasn't been regenerated.